src/Uno.UI-Skia-only.slnf

```diff
@@ -54,7 +54,6 @@
       "Uno.UI.Toolkit\\Uno.UI.Toolkit.Skia.csproj",
       "Uno.UI.XamlHost.Skia.Wpf\\Uno.UI.XamlHost.Skia.Wpf.csproj",
       "Uno.UI.XamlHost\\Uno.UI.XamlHost.Skia.csproj",
-      "Uno.UI\\Uno.UI.Reference.csproj",
```

Comment:
This is supposed to be problematic. Have you tried a local build for the whole solution inside Visual Studio?

Reply:
Oh, it's not yet required on current master, but it will be required by the way https://github.com/unoplatform/uno/pull/18216 is doing the packaging.

Reply:
Then I guess we will need to add Uno.Reference and Uno.Foundation.Reference as well, otherwise VS complains about missing types